crates/hulk_behavior_simulator/src/robot.rs

```diff
@@ -83,6 +83,7 @@ impl Robot {
             .borrow_mut()
             .insert("additional_outputs".to_string());
 
+        database.main_outputs.buttons.is_chest_button_pressed_once = true;
```

Comment:
Please move this up to the other statements in this style, i.e. e.g. line 82

Reply:
Done